### ISSUE TITLE
OpenSSl 

### ISSUE THREAD

**Issue by evgeniy-kolmak** [ssharkexe/telegram-nginx-docker-webhook#2]
А если использовать OpenSSL, а не Let's Encrypt? Можно получается и без домена привязанного к VPS, достаточно будет простого IP.
У меня есть готовый бот но он написан на node, а не на python. Вот думаю завернуть в контейнер что бы все хвосты были внутри. Попытаться как то портировать вашу инструкцию на своего бота. Потому что ничего адекватного больше не нашел.

**Comment by ssharkexe** [comment]
Да, можно и самоподписный подсунуть сертификат, telegram должен его проглотить. Постараюсь в ближайшее время дополнить инструкцию с openssl
